pkgs/games/umoria/default.nix

```diff
@@ -0,0 +1,60 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, autoreconfHook, cmake, ncurses6 }:
+let
+  savesDir = "~/.umoria/";
+in
+gcc9Stdenv.mkDerivation rec {
```

Comment:
It does not build in most recent GCC?

Reply:
In the [official README](https://github.com/dungeons-of-moria/umoria#readme) it's written that it is "tested against GCC 8.x, and 9.3, with ncurses 6.x, although recent earlier versions should also work fine".
I tried compiling it using the current unstable's `stdenv` (GCC 10.0.3) and the compilation failed.
I think it could be easily patched to make it compile with GCC 10, but after reading the README I would keep using `gcc9Stdenv`.